PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetEnergyFlow.h

```diff
@@ -41,7 +41,8 @@ class AliAnalysisTaskEmcalJetEnergyFlow: public AliAnalysisTaskEmcalJet {
 		const char *ntracks		= "usedefault",
 		const char *nclusters		= "usedefault",
 		const char *ncells		= "usedefault",
-		const char *suffix		= "");
+		const char *suffix		= "",
+                Bool_t isMCprod                 = kTRUE               );
```

Comment:
Your suffix should be last if you want to support subwagons

Reply:
Thank you for the valuable feedback! I think that I incorporated the changes you suggested ( and unless I messed up my with my git) the latest PR should include these changes.

Reply:
Thanks, it looks good. Approving now